src/customers/account-dashboard-my-returns.md

```diff
@@ -0,0 +1,57 @@
+---
+title: My Returns
+---
+
+All customer return information is displayed on the **My Returns** page in the customer account.
+
+![]({% link images/images/my-returns-page.png %}){: .zoom}
+_My Returns Page_
+
+## Create Return Request from customer account
+
+Only shipped and invoiced orders, with `Complete` status, can be returned.
+
+1. From the storefront, the customer logs in and opens their customer account page.
+
+1. The customer opens the **My Orders** page and finds the required order with the status `Complete`.
+
+1. Click <span class="btn">View Order</span>.
+
+1. To create a return request, click <span class="btn">Return</span>.
+
+   ![]({% link images/images/my-orders-returns.png %}){: .zoom}
+   _My Orders Page_
+
+1. Fill in the **Contact Email Address** field.
+
+1. Chooses the **Item** to return and enters the **Quantity to Return**.
+
+1. Sets **Resolution** to one of the following:
+
+    - Exchange
+    - Refund
+    - Store Credit
+
+1. Sets **Item Condition** to one of the following:
+
+    - Unopened
+    - Opened
+    - Damaged
+
+1. Sets **Reason to Return** to one of the following:
+
+    - Wrong Color
+    - Wrong Size
+    - Out of Service
+    - Other
+
+    In case of the reason for the return is different from those listed in the **Reason to Return** field, then you can enter your own in the **Other** field.
+
+1. If you need to return more than one item, click <span class="btn">Add item to return</span> and repeat the previous steps.
+
+    ![]({% link images/images/create-new-return.png %}){: .zoom}
+   _Create New Return Page_
+
+1. Also, you can leave an additional comment for the store admin in the **Comment** field.
+
+1. To create a return, click <span class="btn">Submit</span>.
```

Comment:
```suggestion
1. Clicks <span class="btn">Submit</span> to create the return request.
```

Reply:
@leslietilling, hi
thank you very much for all your help!
Done.